scale-tests/uninstall-dispatchers.sh

```diff
@@ -6,39 +6,73 @@ set -x
 # Usage:
 #    (1) Login to cluster via "dcos cluster" command.
 #    (2) Run this script:
-#        ./uninstall-dispatchers.sh [dispatchers_file]
+#        ./uninstall-dispatchers.sh [dispatchers_file] [spark_principal]
 #
 # This script:
 # - uninstalls instances of spark listed in an input file.
 
 if [ "$#" -lt 1 ]; then
-    echo "Usage: ./uninstall-dispatchers.sh [dispatchers_file]"
+    echo "Usage: ./uninstall-dispatchers.sh [dispatchers_file] [spark_principal]"
     exit 1
 fi
 
-ACS_TOKEN=`dcos config show core.dcos_acs_token`
-PACKAGE_NAME=spark
+readonly DISPATCHERS_FILE="${1:-}"
```

Comment:
Since `DISPATCHERS_FILE` is required, should this not just be `${1}`?

Reply:
https://github.com/mesosphere/spark-build/pull/306/commits/a141cb3ab284de217a7e1f58b16011d3ef85d85a